coderd/prebuilds/preset_snapshot.go

```diff
@@ -43,10 +48,11 @@ type PresetSnapshot struct {
 // calculated from a PresetSnapshot. While PresetSnapshot contains raw data,
 // ReconciliationState contains derived metrics that are directly used to
 // determine what actions are needed (create, delete, or backoff).
-// For example, it calculates how many prebuilds are eligible, how many are
-// extraneous, and how many are in various transition states.
+// For example, it calculates how many prebuilds are expired, eligible,
+// how many are extraneous, and how many are in various transition states.
 type ReconciliationState struct {
-	Actual     int32 // Number of currently running prebuilds
+	Actual     int32 // Number of currently valid running prebuilds, i.e., non-expired prebuilds
```

Comment:
Same rename comment as [here](https://github.com/coder/coder/pull/17996/files#r2103205410) applies to `Actual` state 😅